devices/Ld2410/GateConfiguration.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+using UnitsNet;
+
+namespace Iot.Device.Ld2410
+{
+    /// <summary>
+    /// Defines a specific gate's sensitivity configurations.
+    /// </summary>
+    public sealed class GateConfiguration
+    {
+        /// <summary>
+        /// The distance, in centimeters, per each radar distance gate.
+        /// </summary>
+        public readonly static byte DistancePerGateInCm = 75;
```

Comment:
can we get a Length instead of the byte? not everyone is fluent in normal units ;-)